book/src/super-sql/operators/unnest.md

```diff
@@ -52,8 +52,8 @@ sequence has the form:
 where the first field is copied to each derived value and the second field is
 the unnested elements of the array `elem0`, `elem1`, etc.
 
-To explode the fields of records or the key-value paris of maps, use the  
-[`flatten`](../functions/records/flatten.md) function, which produces an array that
+To explode the fields of records or the key-value pairs of maps, use the
+[`flatten` function](../functions/records/flatten.md), which produces an array that
```

Comment:
This is not currently accurate as written since per open issue #6050 `flatten` does not currently support maps. I'd advocate we either commit to fixing that issue before GA release or add a caveat here that support for maps is planned for the future.